### PR TITLE
CB-10594 geolocation.spec.10 crashes Windows Phone 8.1

### PR DESCRIPTION
**Pull request by daserge**
[Jira issue](https://issues.apache.org/jira/browse/CB-10594)

### CODE REVIEW

File: tests/tests.js [hunk 1]
```diff
@@ -174,8 +174,12 @@ exports.defineAutoTests = function () {
...
 
             var errorWatch = null;
-            afterEach(function () {
-                navigator.geolocation.clearWatch(errorWatch);
+            afterEach(function (done) {
+                // Workaround to avoid the crash "0x80004004 - JavaScript runtime error: Operation aborted" on Windows
+                setTimeout(function () {
+                    navigator.geolocation.clearWatch(errorWatch);
+                    done();
+                }, 100);
```
Comment (dblotsky) [inline tests/tests.js]: This looks like magic. Why does 100ms fix the issue?

**Comment by daserge** [comment]
@dblotsky, @sarangan12, I've updated the PR, please take a look.

**Comment by sarangan12** [comment]
@daserge I am not sure if removing the event listeners is a good idea. From the user perspective, there is no reason for him to have only one event listener. All of this boils down to one issue, reusing the 'loc' variable. With the restriction that you cannot modify the properties when there are event listeners on it, I do not see a good point in reusing it (in ensurelocator function) I have created another PR to address that issue: https://github.com/apache/cordova-plugin-geolocation/pull/70

**Comment by daserge** [comment]
@sarangan12, listeners are removed and then subsribed again.

**Comment by dblotsky** [comment]
@daserge this change will change an *existing* listener, which is not expected behaviour. It is expected that when `addWatch` is called a second time, a new watch is created and the previous watch is not modified.

**Comment by dblotsky** [comment]
Please review #70. It looks like it specifically fixes the behaviour to match what's expected.

**Pull request closed without merging**